public/index.html

```diff
@@ -19,7 +19,7 @@
     <title>Marc the magician</title>
   </head>
   <body>
-    <h1>calculator</h1>
+    <div class="helpfull">Calculator app</div>
```

Comment:
[OPTIONAL] There is a bug when trying to use the modulo `%` sign. It could be nice if you can fix it so it works just as the division `/` sign when someone tries to divide by 0.

![image](https://user-images.githubusercontent.com/79114233/141243329-b583edf8-0d53-4fa3-ab16-484ce2ab8871.png)

Reply:
bug fixed